include/cantera/thermo/Elements.h

```diff
@@ -84,6 +84,50 @@ namespace Cantera
 //! stable state at 298.15 K and 1 bar.
 #define ENTROPY298_UNKNOWN -123456789.
 
+//! Get a vector of the atomic symbols of the elements defined in Cantera.
+//! @since New in version 3.0
+const std::vector<std::string>& elementSymbols();
+
+//! Get a vector of the names of the elements defined in Cantera.
+//! @since New in version 3.0
+const std::vector<std::string>& elementNames();
```

Comment:
What advantage does the map provide? I don't think it'd simplify that much, since you still don't know whether you're looking up a symbol or name without trying it first.

The other concern that I have is, what's the key of the map? The atomic number? The weight? The symbol? I don't know unless I look at the docstring. Then would we want maps for everything else? I'm not inclined to implement this at the moment... 😕

Reply:
No worries - mu suggestion would be useful if elements handling is made consistent for the overrides - the Element classes  are currently not aware of these, which makes things quite hard to follow. While this is not being done, the internal `atomicWeightTable` and `isotopeWeightTable` objects (which are not accessible otherwise) are sufficient to handle everything.

Reply:
One option is a [`boost::bimap`](https://www.boost.org/doc/libs/1_79_0/libs/bimap/doc/html/boost_bimap/reference.html#boost_bimap.reference.headers) which is header only, I think!

Reply:
That's an interesting option.